dolphinscheduler-dist/src/main/provisio/dolphinscheduler.xml

```diff
@@ -131,4 +131,9 @@
             <unpack/>
         </artifact>
     </artifactSet>
+    <artifactSet to="lib/plugin/task/dataquality">
```

Comment:
If this is a task plugin, why don't you move the module to `dolphinscheduler-task-plugin/dolphinscheduler-data-quality`?

Reply:
All plugin artifactset write in this file，no need move to data-quality-plugin module